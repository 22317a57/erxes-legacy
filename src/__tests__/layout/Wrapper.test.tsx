import { mount, shallow } from 'enzyme';
import React from 'react';

import Wrapper from '../../modules/layout/components/Wrapper';

describe('WithPermission component', () => {
  const defaultProps = {
    content: 'content'
  };

  test('renders Wrapper successfully', () => {
    shallow(<Wrapper {...defaultProps} />);
  });

  test('renders successfully with default value', () => {
    const wrapper = mount(<Wrapper {...defaultProps} />);
    const props = wrapper.props();

    expect(props).toMatchObject(defaultProps);
  });
});
