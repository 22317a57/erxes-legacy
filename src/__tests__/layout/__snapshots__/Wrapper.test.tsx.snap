// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithPermission component snapshot matches 1`] = `
<div
  className="sc-cSHVUG iARDhv"
>
  <section
    className="sc-kAzzGY eWxWDz"
  >
    <div
      className="sc-chPdSV kDipgy"
    >
      content
    </div>
  </section>
</div>
`;
